src/main/java/sawkiewicz/task1/ItemController.java

```diff
@@ -0,0 +1,38 @@
+
+package sawkiewicz.task1;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+
+@RestController
+public class ItemController {
+
+    List<Item> items = ItemRepository.getItems();
```

Comment:
new ItemRepository()

Reply:
`getItems()` should be called at the beginning of every methods that uses `items` because underlying datastore can be changed